src/main/java/uk/gov/hmcts/dts/fact/mapit/MapItService.java

```diff
@@ -47,4 +56,15 @@ private void validateQuotaLimit(final JsonNode quota) {
             throw new MapitUsageException();
         }
     }
+
+    /**
+     * Accepts an array of strings and checks for each if mapit data exists.
+     * @param postcodes An array of strings which are postcodes
+     * @return An array of strings which indicate which postcodes have failed to return geographical information
+     */
+    public String[] validatePostcodes(String... postcodes) {
+        return Arrays.stream(postcodes)
+            .filter(postcode -> !mapitValidater.postcodeDataExists(postcode.trim().replaceAll("\\s+","")))
```

Comment:
Don't think we need the trim() method here. The replaceAll() should handle all spaces include the preceding and trailing spaces